src/integrationTest/java/uk/gov/hmcts/darts/cases/service/CaseServiceAdvancedSearchTest.java

```diff
@@ -307,6 +309,42 @@ void getWithCourtroomJudge() throws IOException {
         compareJson(actualResponse, expectedResponse);
     }
 
+    @Test
+    void whenAdvancedSearchIsRunForUserWithGlobalAccessWithNoCourthouseAccess_thenShouldReturnResultsOk() throws IOException {
+
+        GetCasesSearchRequest request = GetCasesSearchRequest.builder()
+            .caseNumber("sE1")
+            .build();
+
+        UserAccountEntity testUser = dartsDatabase.getUserAccountStub().createAuthorisedIntegrationTestUserWithoutCourthouse();
+        when(mockUserIdentity.getUserAccount()).thenReturn(testUser);
+        when(mockUserIdentity.userHasGlobalAccess(Set.of(JUDGE))).thenReturn(true);
+
+        List<AdvancedSearchResult> resultList = service.advancedSearch(request);
+        String actualResponse = TestUtils.removeIds(objectMapper.writeValueAsString(resultList));
+        String expectedResponse = TestUtils.removeIds(getContentsFromFile(
+            "tests/cases/CaseServiceAdvancedSearchTest/getWithCaseNumber/expectedResponse.json"));
+
+        compareJson(actualResponse, expectedResponse);
+    }
+
+    @Test
+    void whenAdvancedSearchIsRunForUserWithoutGlobalAccessWithNoCourthouseAccess_thenShouldReturnEmptyArray() throws IOException {
```

Comment:
Can you add another test that tests that when a user is linked via their security groups to a courthouse that it still works?